README.md

```diff
@@ -92,79 +92,91 @@ end program simple_get
 ```
 In this example, we make a GET request to the URL https://jsonplaceholder.typicode.com/todos/1 to retrieve JSON data. If the request is successful, we print the ***response code, content length, method, and content***. If the request fails, we print the ***error message***.
 
-## **Getting Started Guides**
-> ### ***The Complete Tutorial Guide is <u> Currently in progress</u> and will be available soon.***
-1. ### **Installation** 👋
-    - Installing Dependencies (Ubuntu)
-    - Setting up the Package in your Project
-
-2. ### **Making HTTP Requests** 🚀
-    - **Sending `GET` Requests**
-        - *Accessing Response `Content`*
-        - *Retrieving `Status Codes`*
-        - *Getting Response `Headers`*
-        - *Extracting `Content Length`*
-    - **Sending `POST` Requests**
-        - *Sending `Data` with Requests*
-        - *Sending `Form Data`*
-        - *Uploading `File`*
-    - **Sending `PUT` Requests**
-    - **Sending `PATCH` Requests**
-    - **Sending `DELETE` Requests**
-    - **Sending `HEAD` Requests**
-
-3. ### **Customizing Requests** ✏️
-    - Sending Custom **Headers**
-    - Setting Request **Timeout**
-    - **Authentication** Option
-
-4. ### **Error Handling** 🤨
-    - Handling Unsuccessful Requests
-    - Displaying Error Messages
-
-5. ### **Real Projects** 🤖
-    -  **GitHub organization analyzer** : Retrieve valuable information about the organization repositories
-
-
-## **Contributing to project**
+## Getting Started Guides 👋
+
+
+### Table of contents: 
```

Comment:
Rather than having the tutorial TOC here, why not put it on top of tutorial.md? It seems more appropriate that a TOC for a doc would be in that doc, not outside of it. Here in the README you can just link to the tutorial doc.